### PR TITLE
[#166949102] Programmatically add io.italia.it dns zone and records

### PR DESCRIPTION
**Pull request by LucaPrete**

### CODE REVIEW

File: azurerm_dns_zone_records_io_italia_it/variables.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  description = "The data center location where all resources will be put into."
+}
+
+variable "resource_name_prefix" {
+  description = "The prefix used to name all resources created."
+}
+
+# DNS module specific variables
+
+variable "azurerm_public_ip_name" {
```
Comment (l-pinna) [inline azurerm_dns_zone_records_io_italia_it/variables.tf]: Description should be checked (seems a refuse)

File: azurerm_dns_zone_records_io_italia_it/variables.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  type        = "list"
+}
+
+variable "mailup_cname_records" {
+  description = "The list of dictionaries with the name and the value of the MailUp related CNAME records."
+  type        = "list"
+}
+
+locals {
+  azurerm_public_ip_name                     = "${var.resource_name_prefix}-${var.environment}-pip-${var.azurerm_public_ip_name}"
```
Comment (l-pinna) [inline azurerm_dns_zone_records_io_italia_it/variables.tf]: where is used?

File: azurerm_dns_zone_records_io_italia_it/variables.tf (hunk 1 above)
Comment (LucaPrete) [reply azurerm_dns_zone_records_io_italia_it/variables.tf]: Seems correct to me. I've switched the order of name and suffix

File: azurerm_dns_zone_records_io_italia_it/variables.tf (hunk 2 above)
Comment (LucaPrete) [reply azurerm_dns_zone_records_io_italia_it/variables.tf]: in the main file....exactly under the object azurerm_public_ip - param name

**Pull request merged**